test/integration/extension/lib/bigcommerce/CartRepositoryTest.js

```diff
@@ -50,10 +53,20 @@ describe('BigCommerceCartRepository - integration', () => {
     })
   })
 
-  it('should provide checkout url', function () {
+  it('should provide checkout url', function (done) {
     storageMock.expects('get').once().returns(cartId)
 
-    return subjectUnderTest.getCheckoutUrl().should.eventually.be.a.string
+    // The BigCommerce API call getCheckoutUrl was sometimes throwing an error.
+    // We need the timeout to give the BigCommerce infrastructure time to replicate their data along all instances
+    setTimeout(async () => {
```

Comment:
There is a cleaner way to deal with timeouts, see:
* https://stackoverflow.com/questions/15971167/how-to-increase-timeout-for-a-single-test-case-in-mocha
* https://mochajs.org/#test-level

Reply:
The first link is unrelated to the setTimeout code. Both links just show how you can increase the timeout of a test but I want to delay a test. If you wanted to emphasize something different from what I found on both pages please be more concrete.

I can change the description of the comment because "we don't need the timeout" is misleading. We need a delay. Maybe thats the misunderstanding here.